src/site/apt/index.apt.vm

```diff
@@ -29,8 +29,13 @@
 ${project.name}
 
   The Compiler Plugin is used to compile the sources of your project. The
-  default compiler used to compile Java sources is javax.tools.JavaCompiler.
-  Other compilers than <<<javac>>> can be used.
+  default compiler used to compile Java sources is <<<javac>>>.
+  If you want to use another compiler, refer to the {{{./non-javac-compilers} guide: Using Non-Javac Compilers}}.
```

Comment:
Can we use non javac compilers after: #271 @gnodet @desruisseaux ?

Reply:
Yes. Any compiler that implements the `javax.tool.JavaCompiler` interface should work. It is the case of Eclipse compiler for instance. I'm not sure if it has been tested however (not on my side), but we will fix issues or report them to ECJ if we found some.

Reply:
> So should we update `non-javac-compilers` guide?

And shall I do it in this PR or do you (@desruisseaux) want to do it, when "the work on sources" (see other comment) is done?

Reply:
non-javac-compilers guide - in next PR, the best with IT

Reply:
> non-javac-compilers guide - in next PR, the best with IT

created https://issues.apache.org/jira/browse/MCOMPILER-609 for this.